designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/Text.kt

```diff
@@ -133,10 +133,14 @@ internal fun ListCardTitle(
     title: ListCardTitleModel,
     modifier: Modifier = Modifier,
 ) {
+    val maxLines = if (title.allowOverflow) Int.MAX_VALUE else 2
```

Comment:
Can we add some test and or and example in the showcase for testing and validation?

Reply:
Is it worth it to use a constant rather than a simple 2??